MIGRATING.MD

```diff
@@ -64,32 +79,37 @@ OpenTelemetry Java Instrumentation.
 ### Configuration setting changes
 
 These SignalFx Java Agent system properties correspond to the following
-OpenTelemetry system properties:
+OpenTelemetry system properties (NOTE: for exporter - specific properties we use `jaeger` as an example):
```

Comment:
Should we even mention other exporters when jaeger is the default?

Reply:
I think so - if the customer selects other exporter suggested config change `SIGNALFX_ENDPOINT_URL` ->|`OTEL_EXPORTER_JAEGER_ENDPOINT` will not work.